src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IHandlerContext.kt

```diff
@@ -33,6 +33,11 @@ interface IHandlerContext {
      */
     val settings: IHandlerSettings
 
+    /**
+     * Returns book name for this service
+     */
+    val bookName: String
```

Comment:
Why do we need it here?

Reply:
Where is the better place for this? We need to access that from handler when there is data-provider communication.

Reply:
> We need to access that from handler when there is data-provider communication

That's what I thought, thanks. Ok, let's keep it here